test/TinyRAM/Spec/EndToEndSpec (copy).hs

```diff
@@ -0,0 +1,509 @@
+{-# LANGUAGE NoImplicitPrelude   #-}
```

Comment:
This file has a (copy) in its name and, I supposed, was added to source control by accident. Please remove this file and replace content of EndToEndSpec.hs with it.